gemfiles/rails_3.2.gemfile

```diff
@@ -3,7 +3,8 @@
 source "http://www.rubygems.org"
 
 gem "appraisal"
+gem "sqlite3", "~> 1.3.6"
 gem "activerecord", "~> 3.2.22"
-gem 'minitest' # To fix ruby 2.2 + rails 3.2 build
+gem "minitest"
```

Comment:
Can you keep the comment?

Reply:
Unfortunately I can't keep the comment and still keep the build passing using only the output from `appraisal generate`. I could make it clearer why this line exists in the commit message though, so that a `git blame` will give an even quicker answer?